push-helm-charts.sh

```diff
@@ -13,38 +13,38 @@ REPO_DIR=bin/charts # This is where we do the charge merge and dirty things up,
 mkdir -p $REPO_DIR
 echo "entering $REPO_DIR"
 cd $REPO_DIR
-if curl --output /dev/null --silent --head --fail ${CHARTS_REPO}/${CHARTS_BUCKET}/index.yaml; then
+if curl --output /dev/null --silent --head --fail "$CHARTS_REPO/$CHARTS_BUCKET/index.yaml"; then
   echo "downloading existing index.yaml"
-  curl -sLO ${CHARTS_REPO}/${CHARTS_BUCKET}/index.yaml
+  curl -sLO "$CHARTS_REPO/$CHARTS_BUCKET/index.yaml"
 fi
 
- #####
+#####
 # package the charts
 #####
-for dir in `ls ../../charts`;do
-    if [ ! -f ../../charts/$dir/Chart.yaml ];then
-        echo "skipping $dir because it lacks a Chart.yaml file"
-    else
-        echo "packaging $dir"
-        helm dep build ../../charts/$dir
-        helm package ../../charts/$dir
-    fi
+for dir in charts/*; do
+  if [ ! -f "../../charts/$dir/Chart.yaml" ]; then
+    echo "skipping $dir because it lacks a Chart.yaml file"
+  else
+    echo "packaging $dir"
+    helm dep build "../../charts/$dir"
+    helm package "../../charts/$dir"
+  fi
 done
 
- if [ -f $REPO_DIR/index.yaml ]; then
+if [ -f ./index.yaml ]; then
```

Comment:
Just to help pick out the fix, this was it - at this point you're still in `$REPO_DIR`, so previously it would be look in `bin/charts/bin/charts` for an `index.yaml`.

The rest of the file was fixing mixed indentation, random indentation, and shellcheck issues.